.github/workflows/builder_nodejs_slsa3.yml

```diff
@@ -46,55 +34,27 @@ on:
         required: false
         type: string
 
-      dist-tag:
-        description: "The package dist-tag to attach. See `npm help dist-tag` for more information."
-        required: false
-        type: string
-        default: "latest"
-
-      upload-assets:
-        description: >
-          If true, provenance is uploaded to a GitHub release for new tags.
-          When the workflow does not run on a new tag, such as on a `workflow_dispatch`,
-          the `upload-tag-name` argument must be provided as well.
-        required: false
-        type: boolean
-        default: true
-
-      upload-tag-name:
-        description: >
-          If non-empty and `upload-assets` is set to true, the provenance is uploaded to the GitHub
-          release identified by the tag name. If a workflow is run on a new tag and `upload-tag-name`
-          is non-empty, the new tag is ignored and the value of `upload-tag-name` is used instead to upload
-          the assets.
-        type: string
-        default: ""
-
     secrets:
```

Comment:
we no longer need the auth token, I think. For private repos, the GITHUB_TOKEN is available so we should still be able to checkout the repo.